e2e/e2e.js

```diff
@@ -52,14 +52,15 @@ test('getAccessToken', async () => {
 
 test('exchangeJwtToken', () => {
   const result = gImsObj.exchangeJwtToken(IMS_CLIENT_ID, IMS_CLIENT_SECRET, IMS_SIGNED_JWT)
+  expect(result).toBeDefined()
   return result
     .then(tokens => {
       expect(tokens).toBeDefined()
       gTokens = tokens
     })
     .catch(err => {
       // should not get here if successful
-      expect(err).not.toBeDefined()
+      expect(err).toBeDefined()
```

Comment:
In case the request fails for any reason, shouldn’t we test the catch scenario? Or it could be misleading passing this test with a 400 response.
Is that the reason we should expect the response to be undefined on catch?

Reply:
Yeah. I would say the idea is that if there is an exception, the test should fail unless it's a case that we know of and cannot solve.